### PR TITLE
Medical glasses

### PR DESCRIPTION
**Pull request by DenisShvalov**
## About the PR
Added medical glasses - medical hud in form of glasses

## Why / Balance
I like glasses
Also, added medical hud and glasses craft in medical techfab

## Media

https://github.com/space-wizards/space-station-14/assets/115770678/447c1e18-4849-48f2-b2eb-b7e390d5e673

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

image

**Changelog**

:cl:
- add: Medical glasses - another form of medical hud
- tweak: Added medical glasses and hud recipe in medical techfab

### CODE REVIEW

**Comment by EmoGarbage404** [comment]
ngl leading with the reskin and not with the fact that you're adding recipes to the techfab is a mood

File: Resources/Prototypes/Recipes/Lathes/medical.yml [hunk 1]
```diff
@@ -242,4 +242,20 @@
...
+  materials:
+    Steel: 600
+    Glass: 400
+    
+- type: latheRecipe
+  id: ClothingEyesGlassesMedical
+  result: ClothingEyesGlassesMedical
+  completetime: 2
+  materials:
+    Steel: 400
```
Comment (TheShuEd) [inline Resources/Prototypes/Recipes/Lathes/medical.yml:259]: if it's just a reskin they should have the same material cost

File: Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml [hunk 2]
```diff
@@ -217,3 +217,23 @@
...
+  description: Styled medical hud
+  components:
+  - type: Sprite
+    sprite: Clothing/Eyes/Glasses/medglasses.rsi
+  - type: Clothing
+    sprite: Clothing/Eyes/Glasses/medglasses.rsi
+  - type: ShowHealthBars
+    damageContainers: 
+    - Biological
+  - type: ShowHealthIcons
```
Comment (TheShuEd) [inline Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml:234]: just inherit from the existing medical visor, and don't add duplicate components

**Review by TheShuEd — CHANGES_REQUESTED** [review]

**Comment by TheShuEd** [comment]
I'm still not sure if we need to add resprites to techfab.  therefore not an approval, but simply a cancellation of the request

**Comment by TheShuEd** [comment]
how about this suggestion. Make these glasses look expensive and make them a one-of-a-kind in a medical vending machine. Also add them to the thief's stealing tasks, he has just a shortage of unique items to steal in the science and medical departments.

**Comment by AJCM-git** [comment]
Re-skins like this shouldnt be added without lathes properly handling them (the ui should have a submenu or something to change the sprite to an alternative and have the same recipe or something), if you or someone ends up doing that feel free to reopen

**Pull request closed without merging**